test/parallel/test-net-write-slow.js

```diff
@@ -26,7 +26,7 @@ var server = net.createServer(function(socket) {
   }
   socket.end();
 
-}).listen(0, function() {
+}).listen(0, common.mustCall(function() {
```

Comment:
Not wrong, just noting that in other tests you don't seem to wrap the connection listener in a common.mustCall.